relay-server/src/services/processor.rs

```diff
@@ -798,6 +798,9 @@ struct ProcessEnvelopeState<'a, Group> {
     /// The state of the project that this envelope belongs to.
     project_state: Arc<ProjectInfo>,
 
+    /// The config of this Relay instance.
+    config: Arc<Config>,
```

Comment:
We did not have the config on the state but passed it into all the processing functions which needed it.

For example: https://github.com/getsentry/relay/blob/77ffd711852465aa1c3731b5827e7e9eec13af2d/relay-server/src/services/processor.rs#L1686-L1690

Do we want to replace all of these now as well?

Reply:
Yeah good idea.